lib/rules/no-mixed-spaces-and-tabs.js

```diff
@@ -54,9 +101,19 @@ module.exports = function(context) {
                 match = regex.exec(line);
 
                 if (match) {
+                    var lineNumber = i + 1,
+                        column = match.index + 1;
+
+                    // make sure this isn't inside of a template element
+                    for (; lastTemplateLocIndex < templateLocs.length; lastTemplateLocIndex++) {
+
+                        if (inLoc(templateLocs[lastTemplateLocIndex], lineNumber, column)) {
```

Comment:
What is you have a template literal and a mixed tabs and spacing on the same line? is that too obtuse. Is there a test for that case?